### ISSUE TITLE
DNSD fails to start when using network /8 on interface

### ISSUE THREAD

**Issue by alandrecm** [magma/magma#498]
When using an IP address with network mask 255.0.0.0 on the S1 interface, the service dnsd hangs and do not start. Once you change the mask to 255.255.255.0 it starts normally. 

The log doesn't give much information :
```
Jul 30 21:25:05 magma-agw systemd[1]: Starting Magma dnsd service...
Jul 30 21:26:35 magma-agw systemd[1]: magma@dnsd.service: Start-pre operation timed out. Terminating.
Jul 30 21:26:35 magma-agw systemd[1]: Failed to start Magma dnsd service.
Jul 30 21:26:35 magma-agw systemd[1]: magma@dnsd.service: Unit entered failed state.
Jul 30 21:26:35 magma-agw systemd[1]: magma@dnsd.service: Failed with result 'timeout'.
Jul 30 21:26:40 magma-agw systemd[1]: magma@dnsd.service: Service hold-off time over, scheduling restart.
Jul 30 21:26:40 magma-agw systemd[1]: Stopped Magma dnsd service.
Jul 30 21:26:40 magma-agw systemd[1]: Starting Magma dnsd service...

dnsd.yml & gateway.mconfig: [files.zip](https://github.com/facebookincubator/magma/files/3452965/files.zip)
```

**Comment by annepak88** [comment]
@ssanadhya  - please re-assign if you're not the right person to address this issue.

**Comment by koolzz** [comment]
Resolved by caa651112405107cdbaafb9640cf959a8d27379e
